src/components/call-to-action/index.njk

```diff
@@ -6,20 +6,43 @@ synonyms: cta
 +++
 {% from "views/partials/example/_macro.njk" import patternlibExample %}
 {% from "components/external-link/_macro.njk" import onsExternalLink %}
-{% from "components/panel/_macro.njk" import onsPanel %}
 +++
 
+
+### Call to action
+The call to action button should be used when a link to a new page is the primary call to action on a page. For example, a “Start survey” link.
+
+Links simply styled as buttons are not accessible to users who navigate a web page with voice commands. Setting `"url"` will add the attribute `role="button"` which will help these users select the link.
+{{
+    patternlibExample({"path": "components/call-to-action/examples/button-link/index.njk"})
+}}
+
+#### Call to action opening in a new tab or window
+Use this button when a page needs to open in a new tab or window, for example, for a web chat service.
+
+Setting the `"newWindow"` to `true` adds a warning for screen reader users (set by `"newWindowDescription"`) to let them know a new tab or window will open when they select the button. This helps screen reader users understand why the back button in the browser is disabled on the new page.
 {{
-    onsPanel({
-        "body": "This component requires documentation."
+    patternlibExample({"path": "components/call-to-action/examples/button-link-new-window/index.njk"})
+}}
+
+### Ghost
+The ghosted button style is for use in the [header](/components/header/#external-with-language-and-save-button) or [footer](/components/footer). This can be used to allow a user to exit or [save and sign out](/patterns/save-and-sign-out) of a service.
+
+Setting `"variants"` with `'ghost'` will style the button, and setting `iconType` with `exit` adds the appropriate icon for a “sign out” or “exit” button.
+{{
+    patternlibExample({
+        "path": "components/call-to-action/examples/button-ghost/index.njk",
+        "backgroundColor": "#206095"
     })
 }}
 
-## Default
+### Download
```

Comment:
This one is an awkward fit. Because I don't think users will conceptualise downloading as file navigating away from the current page.

Reply:
So this is an issue of decoupling and naming I think. A download link and even a save and sign out button isn't a CTA. 

Perhaps everything in the docs could live under "button" but the code is split and we just call it 'link-button'

Reply:
I don't think the code and docs need to be the same. In the /button/ folder we could just have `_macro.njk` and `_link-button-macro.njk`

Reply:
In the short term let's keep the docs in 'Button'. I'm going to look at redesigning CTAs/link-buttons to be more "materially honest". Maybe call this macro `_action-link.njk` as the design will likely deviate away from looking like a button.